src/lib/with-layout.js

```diff
@@ -103,6 +115,9 @@ export default function withLayout (Component) {
           toggleDropDown={this.toggleDropDown}
           showClass={this.state.toggleShowClass}
           loggedIn={loggedIn}
+          deleteItem={this.deleteItem}
+          toggleMiniBag={this.toggleMiniBag}
+          minibagDisplayed={this.state.minibagDisplayed}
```

Comment:
I am not very keen on the way `minibagDisplayed` is passed to the layout and then to the actual `MiniBag`. We currently pass it in in two ways:
* as a direct prop here,
* as part of `cart.minibagDisplayed`.

Then we need to check both [here](https://github.com/shiftcommerce/shift-react-components/blob/develop/src/components/layout/minibag.js#L83).

Instead of spreading this piece of information across two separate variables it would be nice to keep it in one place, which could be done here:
```
minibagDisplayed={cart.minibagDisplayed || this.state.minibagDisplayed}
```

Alternatively, we could ditch local state altogether and rely on Redux across the board.

Either would require a follow-up PR to shift-react-components though so I'll leave the decision to you.

Reply:
👍